cumulus/parachains/runtimes/coretime/coretime-rococo/src/xcm_config.rs

```diff
@@ -144,7 +144,7 @@ impl Contains<Location> for ParentOrParentsPlurality {
 }
 
 pub type Barrier = TrailingSetTopicAsId<
-	DenyThenTry<
+	DenyNestedLocalInstructionsThenTry<
```

Comment:
as I wrote before, we don't need type alias `DenyNestedLocalInstructionsThenTry`

```suggestion
	DenyThenTry<
		DenyLocalInstructions<
```

Reply:
Changed to `DenyThenTry<DenyLocalInstructions<Deny, Allow>>`